packages/forms-web-app/src/lib/choose-appropriate-task-list.js

```diff
@@ -21,22 +21,22 @@ const chooseAppropriateTaskList = (appeal) => {
       if (applicationDecision === APPLICATION_DECISION.REFUSED) {
         return pages.householderPlanningTaskList;
       }
-      return pages.fullAppealTaskList;
+      return pages.fullAppeal;
```

Comment:
this function - and the filename - should probably be changed to something more appropriate, as the choice is no longer exclusively between task list pages.

Reply:
It could be renamed to choose-appropriate-application-number-page as it depends on the application type. These conditionals come straight from the business requirements.